### DISCUSSION TITLE
Can we have a discussion board for latest techniques or repos?

### DISCUSSION THREAD

**Discussion by GOOGLE-M** [community/community#19920] [Discussions]
I am wondering whether GitHub can provide a discussion page for latest techniques or repos so that we developers can know what is popular in the tech world more easier.

**Comment by diegoleondev** [comment]
You can visit these GitHub sites, where it shows the topics and trends:

- [Browse popular topics on GitHub](https://github.com/topics)
- [See what the GitHub community is most excited about today](https://github.com/trending)
